official/recommendation/uplift/metrics/metric_configs.py

```diff
@@ -16,12 +16,12 @@
 
 from collections.abc import Mapping
 import dataclasses
-from typing import Any
+from typing import Any, Optional
 
 from official.core.config_definitions import base_config
 
 
-@dataclasses.dataclass(kw_only=True)
+@dataclasses.dataclass
```

Comment:
Is there a specific reason to remove `kw_only=True`?

Reply:
To keep the code more flexible, simple by allowing positional arguments to define attributes for SlicedMetricConfig.

Reply:
Do you have a use case for this? Please do not change this if this change is not a must.

Reply:
I restored the `kw_only=True` as suggested.